ui/hooks/gasFeeInput/useTransactionFunction.test.js

```diff
@@ -90,6 +90,31 @@ describe('useMaxPriorityFeePerGasInput', () => {
     });
   });
 
+  it('should invoke action updateTransaction with 10% increased max priority fee and medium fee + 10% when updateTransactionToTenPercentIncreasedGasFee callback is invoked while original priority fee is 0', async () => {
```

Comment:
We recently decided not to use 'should' anymore. 
```suggestion
  it('invokes action updateTransaction with 10% increased max priority fee and medium fee + 10% when updateTransactionToTenPercentIncreasedGasFee callback is invoked while original priority fee is 0', async () => {
```